pkg/game/test/game.go

```diff
@@ -0,0 +1,154 @@
+package test
+
+import (
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/game/elements"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/game/position"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles/feature"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles/side"
+)
+
+// equivalent to Game type from game package
+type Game interface {
+	GetCurrentTile() (tiles.Tile, error)
+	CurrentPlayer() elements.Player
+	PlayTurn(move elements.PlacedTile) error
+	GetPlayerByID(playerID elements.ID) elements.Player
+	GetBoard() elements.Board
+}
+
+type T interface {
+	Fatal(args ...any)
+	Fatalf(format string, args ...any)
+}
+
+type MeepleParams struct {
+	MeepleType  elements.MeepleType
+	FeatureSide side.Side
+	FeatureType feature.Type
+}
+
+func NoneMeeple() MeepleParams {
+	return MeepleParams{
+		MeepleType:  elements.NoneMeeple,
+		FeatureSide: side.NoSide,
+		FeatureType: feature.NoneType,
+	}
+}
+
+type MakeTurn struct {
+	Game         Game
+	TestingT     T
+	TilePosition position.Position
```

Comment:
Not hugely important but some utilities use `TilePosition` while others just `Position`. I'd suggest using just `Position`:
```suggestion
	Position position.Position
```

Reply:
renamed